docs/markdown/Release-notes-for-0.48.0.md

```diff
@@ -11,6 +11,15 @@ Notable new features should come with release note updates. This is
 done by creating a file snippet called `snippets/featurename.md` and
 whose contents should look like this:
 
+## More flexible `override_find_program()`.
+
+It is now possible to pass an `executable` to
+`override_find_program()` if the overridden program is not used during
+configure.
+
+This is particularly useful for fallback dependencies like protobuf
+that also provide a tool like protoc.
+
```

Comment:
This should not be in this file. It should be in the `snippets` directory as described in the chapter directly above this one.